package/MDAnalysis/converters/RDKitParser.py

```diff
@@ -22,15 +22,15 @@
 #
 
 """
-RDKit topology parser
-=====================
+RDKit topology parser --- :mod:`MDAnalysis.converters.RDKitParser`
```

Comment:
Added these `:mod:` entries since they were present in the non `Parser` modules.